tensorboard/webapp/metrics/views/card_renderer/image_card_test.ts

```diff
@@ -726,42 +705,6 @@ describe('image card', () => {
         );
       });
 
-      it('renders range slider on selected steps which end step is out of range, ', () => {
```

Comment:
Same here. What would be image range slider looks like in this PR?

I'm also wondering any screenshot changes. (We should have tests for image in range)

Reply:
I think I misread this test originally.